.github/workflows/CI.yml

```diff
@@ -0,0 +1,85 @@
+name: CI
+
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - "*"
+
+jobs:
+  lint:
+    name: Lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 10
```

Comment:
shall we use node 12 for this?

Reply:
I copied this from the hardhat repo, we can't modify it. For reference:

- Node 10 will reach end of life in the coming weeks
- Node 11 and 13 are not maintained anymore
- Node 15 reaches end of life on June 1st